lib/galaxy/model/__init__.py

```diff
@@ -4111,10 +4111,12 @@ def _calculate_size(self) -> int:
         return self.object_store.size(self)
 
     @overload
-    def get_size(self, nice_size: Literal[False], calculate_size: bool = True) -> int: ...
+    def get_size(self, nice_size: Literal[False], calculate_size: bool = True) -> int:
```

Comment:
Make sure you got the current dev-dependencies installed before running `make format`. This is the previous black style.